rustybeer-cli/src/commands/calories.rs

```diff
@@ -0,0 +1,90 @@
+use clap::{value_t, App, Arg, ArgMatches};
+pub use rustybeer::calculators::calorie_counter::{
+    calculate_alcohol_calories, calculate_carbs_calories, calculate_total_calories,
+};
+pub use rustybeer_util::abv_calories::{ABVCalories, Criteria, ABV_CALORIES};
+pub use rustybeer_util::conversions::{MassBuilder, VolumeBuilder};
+
+pub fn add_subcommand<'a, 'b>() -> App<'a, 'b> {
+    App::new("calories")
+            .version("0.1")
+            .author("Roger Yu (roger.yu27 [at] gmail.com)")
+            .about("Calculates calories by volume from original and final gravity or from alcohol by volume")
+            .arg(Arg::with_name("og")
+                 .short("o")
+                 .long("og")
+                 .value_name("OG")
+                 .help("Original gravity")
+                 .required(false)
+                 .takes_value(true))
+            .arg(Arg::with_name("fg")
+                 .short("f")
+                 .long("fg")
+                 .value_name("FG")
+                 .help("Final gravity")
+                 .required(false)
+                 .takes_value(true))
+            .arg(Arg::with_name("abv")
+                 .short("a")
+                 .long("abv")
+                 .value_name("ABV")
+                 .help("Alcohol by volume")
+                 .required_unless("fg")
+                 .takes_value(true))
+            .arg(Arg::with_name("volume")
+                 .short("v")
+                 .long("volume")
+                 .value_name("vol")
+                 .help("volume as string ('e.g 10m, 4gal')")
+                 .required(false)
+                 .default_value("100ml")
+                 .takes_value(true))
+}
+
+fn matches_to_criteria<'a>(matches: &ArgMatches<'a>) -> Criteria {
+    Criteria {
+        abv: matches.value_of("abv").map(|value| value.parse().unwrap()),
+    }
+}
+
+pub fn do_matches<'a>(matches: &ArgMatches<'a>) {
+    if let Some(matches) = matches.subcommand_matches("calories") {
+        if matches.is_present("fg") && matches.is_present("og") {
+            let fg = value_t!(matches, "fg", f32).unwrap_or_else(|e| e.exit());
+            let og = value_t!(matches, "og", f32).unwrap_or_else(|e| e.exit());
+            let cv = value_t!(matches, "volume", String).unwrap_or_else(|e| e.exit());
+            let conversion = MassBuilder::from_str("12oz").unwrap().as_grams() as f32;
```

Comment:
You should use the volumes of different bottles merged in #54 and show calories for each volume

Reply:
Hi, to clarify. What you want is a table that list out all of the calories for each volume provided by `num_bottles` but if the user also inputs a volume, then it should only output the calories for that particular volume? So they will either get a table of defaults or just a breakdown for their particular volume.

Reply:
So basically calorie count for 1 of each bottle type + optional custom volume

Reply:
When just providing og and fg:
![image](https://user-images.githubusercontent.com/45835736/97344819-a5598800-185f-11eb-8f78-5b095ad0089b.png)

When providing og, fg, and volume:
![image](https://user-images.githubusercontent.com/45835736/97344915-ccb05500-185f-11eb-8f45-26edf156cd5c.png)

When providing abv:
![image](https://user-images.githubusercontent.com/45835736/97344956-db970780-185f-11eb-93d5-7c8e28714f25.png)

When providing abv and volume:
![image](https://user-images.githubusercontent.com/45835736/97345014-ee114100-185f-11eb-96c3-5f6fbaf3c8e0.png)

I hope this works.